### DISCUSSION TITLE
Is it possible to use "<Plug>" commands  to lazyload?

### DISCUSSION THREAD

**Discussion by garymh** [wbthomason/packer.nvim#226] [Q&A] [answered]
Forgive my ignorance on vim and optional loading 😄

Example: I'd like to be able to do something like this:

`use {'ruanyl/vim-gh-line', cmd = {'<Plug>gh-line', '<Plug>gh-line-blame'  }}`

**Comment by wbthomason** [comment, marked as answer]
That should work, but the trick is that `<Plug>` is a fake key for keybindings, not a command. You probably want something like
```lua
use {'ruanyl/vim-gh-line', keys = {'<Plug>(gh-line)', '<Plug>(gh-line-blame)'}}
```

Hope this helps!

**Reply by garymh** [reply]
Thanks for the quick reply!

That doesn't seem to work, though... It produces: 

```lua
-- Keymap lazy-loads
vim.cmd [[noremap <silent> <Plug>(gh-line) <cmd>lua require("packer.load")({'vim-gh-line'}, { keys = "<lt>Plug>(gh-line)", prefix = "" }, _G.packer_plugins)<cr>]]
vim.cmd [[noremap <silent> <Plug>(gh-line-blame) <cmd>lua require("packer.load")({'vim-gh-line'}, { keys = "<lt>Plug>(gh-line-blame)", prefix = "" }, _G.packer_plugins)<cr>]]
```

I wonder if the `<lt>Plug` is causing a problem?

**Reply by garymh** [reply]
Ugh, disregard! I forgot to `PackerInstall` 🤦 

Works great. Thank you again!
